javascript/linked-list/linked-list-kth/kth.test.js

```diff
@@ -0,0 +1,32 @@
+const { kthFromEnd, createLinkedList } = require('./kth');
+
+describe('kthFromEnd', () => {
+  test('should return the kth node from the end', () => {
+    const head = createLinkedList([1, 2, 3, 4, 5, 6, 7]);
+    const kthNode = kthFromEnd(head, 3);
+    expect(kthNode.value).toBe(5);
+  });
```

Comment:
As per the instructions, the last node is 0 from the end, so the node 3 from the end should be 4 not 5.